From 527e5107f7b772f634ded8bd09c24dc78a9f1944 Mon Sep 17 00:00:00 2001
From: hscham <hscham@chromium.org>
Date: Wed, 28 Apr 2021 10:44:52 +0900
Subject: [PATCH] Add include header base/notreached.h

base/notreached.h is indirectly included via trace_event.h which is
replaced by trace_event_stub.h in libchrome.

Change-Id: Idab0744550f84a0994b47a467a4b20e069186a3f
---
 ipc/ipc_message_utils.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ipc/ipc_message_utils.cc b/ipc/ipc_message_utils.cc
index 2c699cbf75dd..ec44a8574eb3 100644
--- a/ipc/ipc_message_utils.cc
+++ b/ipc/ipc_message_utils.cc
@@ -11,6 +11,7 @@
 #include "base/json/json_writer.h"
 #include "base/logging.h"
 #include "base/memory/ptr_util.h"
+#include "base/notreached.h"
 #include "base/strings/nullable_string16.h"
 #include "base/strings/string_number_conversions.h"
 #include "base/strings/utf_string_conversions.h"
-- 
2.31.1.527.g47e6f16901-goog

